supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -487,6 +487,17 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 
 *See also*:
 
+[discrete]
+[[codeready-linux-builder-repository]]
+==== image:images/yes.png[yes] CodeReady Linux Builder (CRB) repository (noun)
+*Description*: The RHEL _CodeReady Linux Builder repository_ provides additional packages for use by developers. Packages included in the CodeReady Linux Builder repository are unsupported.
+
+*Use it*: yes; use it expanded and always mention that packages in this repository are unsupported (even in a release note, for example)
```

Comment:
This field should either just be yes, no, or with caution. Any caveats should be included in the description field above. Can you please move these details above?

Reply:
Sure, thank you for catching this